tests/v2/validation/pipeline/qase/reporter/main.go

```diff
@@ -18,7 +18,7 @@ import (
 )
 
 const (
-	automationSuiteID    = int32(14)
+	automationSuiteID    = int32(554)
```

Comment:
I'm curious about this number. Is there any significance to it? Will it ever need to be changed?

Reply:
Makes sense